include/godot_cpp/core/class_db.hpp

```diff
@@ -116,9 +116,12 @@ class ClassDB {
 	static void _register_class(bool p_virtual = false, bool p_exposed = true, bool p_runtime = false);
 
 	template <typename T>
-	static GDExtensionObjectPtr _create_instance_func(void *data) {
+	static GDExtensionObjectPtr _create_instance_func(void *data, bool p_notify_postinitialize) {
 		if constexpr (!std::is_abstract_v<T>) {
 			T *new_object = memnew(T);
```

Comment:
`#define memnew(m_class) (::godot::_pre_initialize<std::remove_pointer_t<decltype(new ("", "") m_class)>>(), ::godot::_post_initialize(new ("", "") m_class))`
mennew will auto call `_post_initialize`, it may be called twice